src/main/scala/nl/knaw/dans/easy/stage/lib/FOXML.scala

```diff
@@ -22,7 +22,13 @@ import scala.xml.{Elem, NodeSeq, XML}
 object FOXML {
 
   def getDatasetFOXML(ownerId: String, emd: EasyMetadata): String = {
-    val dc = XML.loadString(emd.getDublinCoreMetadata.asXMLString())
+    /*
+     * NOTE: DO NOT USER THE asXMLString method. It uses the platform's default charset, which can lead to unexpected
```

Comment:
Also, we should maybe document this in `nl.knaw.dans.pf.language.emd.EasyMetadata`. Also, we could maybe annotate this `asXMLString()` method with `@Warning` from the [lightweight javac-warning-annotation library](https://github.com/pushtorefresh/javac-warning-annotation).

Both are of course assuming that we have access to the source of `nl.knaw.dans.pf.language.emd.EasyMetadata`...